### DISCUSSION TITLE
Add Missing Feather Icons

### DISCUSSION THREAD

**Discussion by Kurt-von-Laven** [community/community#37073] [Actions]
This is a follow-up on github/docs#10177 in response to:

> If there is a newer icon that you think should be added, we can probably update the supported list.

Here are the most relevant excerpts of the original issue:

> The documentation clearly conveys that `branding.icon` should be a Feather icon, and explicitly lists all 257 icons that may be used, helpfully linking to [the Feather icons website](https://feathericons.com/). Considering the number of options available, it is easy to overlook the fact that the latest version of Feather icons, v4.28.0, offers an additional 29 icons that are not listed in the GitHub Docs. Particularly since the Feather icons website has a search box and displays the icons themselves, it is easy to make the mistake of selecting one of those 29 unsupported icons if you get unlucky. This raises the question of which version of Feather icons is in use, which has no answer. There is no point in the version history of Feather icons that corresponds to the exact list of icons in use, and it's not clear to me why those specific icons were removed beyond the fact that some of them are for specific software products. In light of this, it would be helpful to mention explicitly the 29 unsupported icons or at least the fact that only the listed icons are supported. (Or better yet, to support all 286 icons, and list them all along with the version of Feather icons in use.)

> Here is the list of the 13 missing icons that remain after brand-specific icons removed. I wasn't sure whether pen-tool should also be included since it's a tool in Adobe Illustrator.

> ```
> coffee
> columns
> divide-circle
> divide-square
> divide
> frown
> hexagon
> key
> meh
> mouse-pointer
> smile
> tool
> x-octagon
> ```
> 

> It would be great if those 13 missing icons could be added. Coffee and tool, in particular, would make great logos for community-contributed GitHub Actions. On a side note, is it possible that Facebook was overlooked when removing brand-specific icons?

Feather Icons v4.29.0 has since been released adding the `table` icon and a few more brand-specific icons.

**Comment by ssbarnea** [comment]
Any updates on this?
